dapps/react-dapp-auth/pages/index.tsx

```diff
@@ -60,81 +49,27 @@ const Home: NextPage = () => {
   useEffect(() => {
     if (!client) return;
     client.on("auth_response", (res) => {
-      if (!!res.params.result.s) {
```

Comment:
Removing this would be a regression right? We tweaked this late last week to check for `result.s` specifically I believe